pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -237,6 +243,15 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard, ICo
         return !getMustAllowlistLPs() || _allowedAddresses[member];
     }
 
+    /**
+     * @notice Returns the current value of the swap fee percentage.
+     * @dev Computes the current swap fee percentage, which can change every block if a gradual swap fee
+     * update is in progress.
+     */
+    function getSwapFeePercentage() public view override returns (uint256) {
```

Comment:
```suggestion
    function getSwapFeePercentage() external view override returns (uint256) {
```

Reply:
This is necessary for now due to `_addSwapFeeAmount` and `_subtractSwapFeeAmount`